slidez-fe-addon/src/client/sidebar-demo/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
It's better to give correct names for files, as it seems to be  just a sidebar later, but it FYI